src/mainwindow.cpp

```diff
@@ -160,6 +161,13 @@ MainWindow::MainWindow(QWidget* parent)
     showTimelineAction->setShortcut(tr("Ctrl+T"));
     connect(showTimelineAction, &QAction::toggled, m_resultsPage, &ResultsPage::setTimelineVisible);
 
+    auto *prettifySymbolsAction = ui->viewMenu->addAction(tr("Prettify Symbols"));
+    prettifySymbolsAction->setCheckable(true);
+    prettifySymbolsAction->setChecked(Settings::instance()->prettifySymbols());
+    prettifySymbolsAction->setShortcut(tr("Ctrl+Y"));
```

Comment:
I don't think we need a shortcut for that? If we do, why Ctrl + Y - that's an undo action usually. If you really want a shortcut, please propose an alternative instead.